docs/index.bs

```diff
@@ -1578,6 +1578,7 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
         RunningStatus runningStatus;
 
         sequence&lt;RouterCondition&gt; _or;
+        RouterCondition _not;
```

Comment:
I think this one does not need escaping, so just `not` is OK?

Reply:
done.